style/modules/event.css

```diff
@@ -0,0 +1,106 @@
+.box .event {
```

Comment:
this is a good strategy for not putting certain things on `.event` -- however I'd style something like `.gridBox .event` instead of `.box event`, since `.box` is there for looks rather than doing any laying out.

overall, I think the best approach is to skip setting height or width, but put it inside of `.gridBox.gridBox-col2` in the storybook. that will take care of the width, and it's likely how we will lay it out on the finished page.

Reply:
also, to make the link the height of the image to the width, without actually setting a width, you'll want something like this: https://makandracards.com/makandra/45969-how-to-define-height-of-a-div-as-percentage-of-its-variable-width

Reply:
...then again, that does require the image URLs to be set in the CSS, which isn't great for maintenance. :/

we'll have to look into this further, I guess? I think it'd be nice if these worked at a variety of widths, but I'm not sure how much trouble it's worth going to for.

Reply:
ok. forget what I said earlier :P

I played with it and the style below looks fine at a variety of widths:


```css
.event {
    display: block;
    /* width: 500px; */
    /* height: 720px; */
}

.event--img {
    /* width: 500px; */
    width: 100%;
    height: 300px;
    object-fit: cover;
}
```

Reply:
should i put `<div class="box event">` inside `.gridBox.gridBox-col2` and remove my `event` section?

Reply:
do I just put the whole `<div class="event">` into `.gridBox.gridBox-col2` under `<div class="gridBox gridBox-col2">` ?

Reply:
yeah. sorry if I wasn't clear. put together it should look something like

```html
<section class="demo" data-module-name="event">
    <div class="gridBox gridBox-col2">
        <div class="box">
            <div class="event">
                ...
            </div>
        </div>
        <div class="box">
            <div class="event">
                ...
            </div>
        </div>
    </div>
</section>
```

and in general we should probably start using `gridBox` in the demos if the final page will have it, so that there are no surprises later